chart/values.yaml

```diff
@@ -42,7 +42,7 @@ securityContext: {}
 # Used for mount paths
 airflowHome: /opt/airflow
 
-# Default airflow repository -- overrides all the specific images below
+# Default airflow repository -- overridden by all the specific images below
```

Comment:
Can you undo all the other formatting changes except this one please -- let's us keep the git history better

Reply:
My bad, Had "format on save" turned on. What's the cleanest way to do that?  Undo the last commit on my topic branch, revise and re-push?

Reply:
Yeah :)